doc/man3/SSL_CTX_new.pod

```diff
@@ -70,20 +72,29 @@ functions
 
 =head1 DESCRIPTION
 
-SSL_CTX_new() creates a new B<SSL_CTX> object as framework to
-establish TLS/SSL or DTLS enabled connections. An B<SSL_CTX> object is
-reference counted. Creating an B<SSL_CTX> object for the first time increments
-the reference count. Freeing it (using SSL_CTX_free) decrements it. When the
-reference count drops to zero, any memory or resources allocated to the
-B<SSL_CTX> object are freed. SSL_CTX_up_ref() increments the reference count for
-an existing B<SSL_CTX> structure.
+SSL_CTX_new_ex() creates a new B<SSL_CTX> object as a framework to
+establish TLS/SSL or DTLS enabled connections using the library context
+I<libctx> (see L<OPENSSL_CTX(3)>). Any cryptographic algorithms that are used
+by any SSL objects will be fetched from the I<libctx> using the property query
+string I<propq> (see L<provider(7)/Fetching algorithms>. Either the I<libctx>
+or I<propq> parameters may be NULL. Note that the I<propq> value is not
+duplicated and is assumed to exist for the lifetime of the SSL_CTX object.
```

Comment:
I don't particularly remember us using language like this for memory-lifetime issues, but I'm also not confident that I would remember if we already had something like it.

Reply:
I actually reconsidered this behaviour and removed the sentence entirely. I now duplicate the provided string. Normally I expect these strings will just be literals, but I can't exclude the possibility that in some circumstances they may be dynamically constructed - so probably "duping" the string is a better idea.